sass/util/_layout.scss

```diff
@@ -8,3 +8,6 @@
 @media #{$breakpoint-md} {
   .md-absolute { position: absolute; }
 }
+
+.right-5 { right: 5%; }
```

Comment:
are these still needed? (it looks like they're incorporated into feedback class)